rts/motoko-rts/Cargo.toml

```diff
@@ -38,4 +38,4 @@ panic = "abort"
 [profile.release]
 panic = "abort"
 codegen-units = 1 # makes debugging easier
-opt-level = "s" # optimize for size
+opt-level = 2 # optimize some
```

Comment:
https://github.com/dfinity/canister-profiling/pull/63/commits `opt 3` seems to do a good job!